client/packages/common/src/ui/forms/JsonForms/components/Select.tsx

```diff
@@ -36,13 +43,17 @@ const UIComponent = (props: ControlProps) => {
         <FormLabel sx={{ fontWeight: 'bold' }}>{label}:</FormLabel>
       </Box>
       <Box flexBasis={FORM_INPUT_COLUMN_WIDTH}>
-        <Select
-          sx={{ minWidth: 100 }}
+        <Autocomplete
+          sx={{ '.MuiFormControl-root': { minWidth: '135px' } }}
           options={options}
-          value={data ?? ''}
-          onChange={e => handleChange(path, e.target.value)}
-          error={!!props.errors}
-          helperText={props.errors}
+          value={value}
+          onChange={onChange}
+          clearable={!!props.config?.required}
```

Comment:
```suggestion
          clearable={!props.config?.required}
```

😉

Reply:
ups yes I remember fixing this before, thanks!